Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -69,17 +69,29 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
     struct HINSTANCE__; typedef struct HINSTANCE__ *HINSTANCE;
 
     typedef HWND hsWindowHndl;
+    typedef HWND hsDisplayHndl;
     typedef HINSTANCE hsWindowInst;
     typedef HINSTANCE HMODULE;
     typedef HMODULE hsLibraryHndl;
     typedef long HRESULT;
     typedef void* HANDLE;
 #elif HS_BUILD_FOR_MACOS
```

Comment:
I think this should now be `#elif defined(HS_BUILD_FOR_APPLE)` since it's not macOS-specific?